compiler-core/src/config.rs

```diff
@@ -138,6 +140,19 @@ impl PackageConfig {
             }
         })
     }
+
+    pub fn specify_hidden_modules(&self, modules: Vec<String>) -> Vec<String> {
+        let mut glob_builder = GlobSetBuilder::new();
+        for pattern in self.documentation.hidden_modules.iter() {
+            let _ = glob_builder.add(Glob::new(pattern).expect("invalid pattern!"));
```

Comment:
Can this ever happen? This is user input right? If so we'll need a user facing error message.

Does the glob type implement serde? We could use it directly in the config struct definition